pkg/operator/ceph/csi/spec.go

```diff
@@ -394,11 +405,99 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	return nil
 }
 
-func StopCSIDrivers(namespace string, clientset kubernetes.Interface) error {
-	logger.Warningf("Skipped removing the CSI driver")
+func StopCSIDrivers(namespace string, clientset kubernetes.Interface, ver *version.Info) error {
```

Comment:
Does this handle the case of removing one of the csi drivers if it is just disabled from the operator env var? Or is this only triggered when removing the last cluster? If not, this could be a separate PR in the future though.